Makefile

```diff
@@ -59,6 +59,9 @@ doccheck: generate
 	@cd docs; for doc in *.md; do if [ "$$doc" != "README.md" ] && ! grep -q "$$doc" *.md; then echo "ERROR: No link to documentation file $${doc} detected"; exit 1; fi; done
 	@echo OK
 
+build-docker:
```

Comment:
can we just instead of duplicating this, make this a dependency of `build-local`?